code/modules/awaymissions/maploader/reader.dm

```diff
@@ -164,12 +164,12 @@ var/global/dmm_suite/preloader/_preloader = new
 	log_debug("Loaded map in [stop_watch(watch)]s.")
 	qdel(LM)
 	if(bounds[MAP_MINX] == 1.#INF) // Shouldn't need to check every item
-		log_debug("Min x: bounds[MAP_MINX]")
-		log_debug("Min y: bounds[MAP_MINY]")
-		log_debug("Min z: bounds[MAP_MINZ]")
-		log_debug("Max x: bounds[MAP_MAXX]")
-		log_debug("Max y: bounds[MAP_MAXY]")
-		log_debug("Max z: bounds[MAP_MAXZ]")
+		log_runtime(EXCEPTION("Bad Map bounds."), src, list("Min x: [bounds[MAP_MINX]]",\
+																												"Min y: [bounds[MAP_MINY]]",\
+																												"Min z: [bounds[MAP_MINZ]]",\
+																												"Max x: [bounds[MAP_MAXX]]",\
+																												"Max y: [bounds[MAP_MAXY]]",\
+																												"Max z: [bounds[MAP_MAXZ]]"))
```

Comment:
List definitions don't require newlines to be escaped. Also, wow, that is a *lot* of tabs.